### ISSUE TITLE
Question about releases

### ISSUE THREAD

**Issue by tacomaan** [andyvorld/LGSTrayBattery#99]
Im quite new to how releases work. Will there be regular new release versions to keep up with the master branch? 
Or is there a way that someone can make their own release out of the latest master iteration?

**Comment by andyvorld** [comment]
Yeah, ideally the master branch is a rolling stable for this project. So building it should be fine, I usually self test master for a few days before make a release. I will make a new release soon.

I was also in the middle of creating automated nightly builds, but I still am working on that.
